app/components/service/includes/secrets.html

```diff
@@ -8,7 +8,15 @@
         <select class="form-control" ng-options="secret.Name for secret in secrets" ng-model="newSecret">
           <option selected disabled hidden value="">Select a secret</option>
         </select>
-        <a class="btn btn-default btn-sm" ng-click="addSecret(service, newSecret)">
+        <div class="input-group col-sm-3 input-group-sm" ng-if="applicationState.endpoint.apiVersion >= 1.30 && secretOverrideTarget">
+          Target:
+          <input class="form-control" ng-model="newSecretTarget" placeholder="/path/in/container">
```

Comment:
Need Help to understand something : 
Here newSecretTarget is always undefined, as I display it in console log
If i remove the div (line 11 & 14 => Target is always active), newSecretTarget is OK in console log
Somebody can help me with that ?

Reply:
This is a binding issue, works fine if you use an object.

```html
        <div class="input-group col-sm-3 input-group-sm" ng-if="applicationState.endpoint.apiVersion >= 1.30 && state.addSecret.override">
          Target:
          <input class="form-control" ng-model="state.addSecret.target" placeholder="/path/in/container">
        </div>
        <div class="btn-group btn-group-sm">
          <label class="btn btn-primary" ng-model="state.addSecret.override" uib-btn-radio="false">Default location</label>
          <label class="btn btn-primary" ng-model="state.addSecret.override" uib-btn-radio="true">Override</label>
        </div>
        <a class="btn btn-default btn-sm" ng-click="addSecret(service, newSecret, state.addSecret.target)">
          <i class="fa fa-plus-circle" aria-hidden="true"></i> add secret
        </a>
```

(also need to update the controller accordingly)